lib/rucio/core/did_meta_plugins/mongo_meta.py

```diff
@@ -36,7 +36,15 @@
 
 
 class MongoDidMeta(DidMetaPlugin):
-    def __init__(self, host=None, port=None, db=None, collection=None):
+    def __init__(
+        self,
+        host: "Optional[str]" = None,
+        port: "Optional[int]" = None,
+        db: "Optional[str]" = None,
+        collection: "Optional[str]" = None,
+        user: "Optional[str]" = None,
+        password: "Optional[str]" = None,
```

Comment:
Since these type hints only use built-in types and `Optional` from `typing`, we could move `from typing import Optional` outside of the `if TYPE_CHECKING` block and remove the quotes from these type annotations.

I'm just leaving this as a small note, no need to make any changes, as this is something that we can lint automatically with Ruff. We temporarily disabled the code that handles this because of false positives: https://github.com/rucio/rucio/blob/fb42cda0c8c49ec11a3fe624be2ca17d208887a8/pyproject.toml#L59, but once we enable it again this will be formatted automatically.